app/res/xml/preferences_developer.xml

```diff
@@ -173,4 +173,9 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-enable-certificate-transparency"
         android:title="Certificate Transparency"/>
+    <EditTextPreference
+        android:defaultValue="@string/logs_server_url_protocol"
```

Comment:
Wondering if user ever sees this `defaultValue` as an app should always have the `log_receiver_url` populated ?

Reply:
they see it when `log_receiver_url` is not set, by [default the logs URL is the Form Submission log](https://github.com/dimagi/commcare-android/blob/831747a00d29328ea0aaa886245e86588fc4af84/app/src/org/commcare/tasks/LogSubmissionTask.java#L112)